Decription: adapt code to build with libheif-rs 0.20
 czkawka normally depends on libheif-rs 0.18
Index: czkawka-core/Cargo.toml
===================================================================
--- czkawka-core.orig/Cargo.toml
+++ czkawka-core/Cargo.toml
@@ -149,11 +149,11 @@ version = "0.12.0"
 features = ["image"]
 
 [dependencies.libheif-rs]
-version = "=0.18.0"
+version = "2.3"
 optional = true
 
 [dependencies.libheif-sys]
-version = "=1.14.2"
+version = "5.0"
 optional = true
 
 [dependencies.libraw-rs]
Index: czkawka-core/src/common/image.rs
===================================================================
--- czkawka-core.orig/src/common/image.rs
+++ czkawka-core/src/common/image.rs
@@ -30,8 +30,8 @@ use crate::common;
 use crate::common::consts::{HEIC_EXTENSIONS, IMAGE_RS_EXTENSIONS, IMAGE_RS_SIMILAR_IMAGES_EXTENSIONS, JXL_IMAGE_EXTENSIONS, RAW_IMAGE_EXTENSIONS};
 use crate::common::create_crash_message;
 use crate::helpers::debug_timer::Timer;
-// #[cfg(feature = "heif")]
-// use libheif_rs::LibHeif;
+#[cfg(feature = "heif")]
+use libheif_rs::LibHeif;
 
 pub(crate) fn get_jxl_image(path: &str) -> anyhow::Result<DynamicImage> {
     let file = File::open(path)?;
@@ -91,11 +91,11 @@ pub fn get_dynamic_image_from_path(path:
 
 #[cfg(feature = "heif")]
 pub(crate) fn get_dynamic_image_from_heic(path: &str) -> anyhow::Result<DynamicImage> {
-    // let libheif = LibHeif::new();
+    let libheif = LibHeif::new();
     let im = HeifContext::read_from_file(path)?;
     let handle = im.primary_image_handle()?;
-    // let image = libheif.decode(&handle, ColorSpace::Rgb(RgbChroma::Rgb), None)?; // Enable when using libheif 0.19
-    let image = handle.decode(ColorSpace::Rgb(RgbChroma::Rgb), None)?;
+    let image = libheif.decode(&handle, ColorSpace::Rgb(RgbChroma::Rgb), None)?; // Enable when using libheif 0.19
+    //let image = handle.decode(ColorSpace::Rgb(RgbChroma::Rgb), None)?;
     let width = image.width();
     let height = image.height();
     let planes = image.planes();
